UnityProject/Assets/Scripts/Managers/AdminManager.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using Pipes;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+using Objects.Wallmounts;
+using System.Collections.Concurrent;
+
+public class AdminManager : MonoBehaviour
+{
+	public ItemStorage LocalAdminGhostStorage;
+	public GameObject ItemStorageHolderPrefab;
+
+	private static AdminManager adminManager;
+	public static AdminManager Instance
+	{
+		get
+		{
+			if (!adminManager)
+			{
+				adminManager = FindObjectOfType<AdminManager>();
+			}
+			return adminManager;
+		}
+	}
+
+	public ItemStorage CreateItemSlotStorage()
+	{
+		var holder = Spawn.ServerPrefab(ItemStorageHolderPrefab, null, gameObject.transform);
+		var itemStorage = holder.GameObject.GetComponent<ItemStorage>();
+		return itemStorage;
+	}
+
+	public void SetLocalAdminGhostStorage()
+	{
+
```

Comment:
this is still a wip, right?

Reply:
No, just a leftlover

Reply:
can you delete the leftover so we merge this and test on staging?